bash/dev-watch.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
+nvm install
+npm install
+npm run dev:css &
+npm run dev:js &
```

Comment:
Question for my own understanding/learning: Why the `&` at the end? What does that do?

Reply:
The final `&` means that line executes in the background and allows the script to execute the next line without waiting for the command to complete.

These are tasks that are never going to be completed as they sit and watch for changes indefinitely.